emission/export/export.py

```diff
@@ -1,72 +1,35 @@
-from __future__ import print_function
-from __future__ import unicode_literals
-from __future__ import division
-from __future__ import absolute_import
-# Exports all data for the particular user for the particular day
-# Used for debugging issues with trip and section generation
-from future import standard_library
-standard_library.install_aliases()
-from builtins import *
-import sys
 import logging
 logging.basicConfig(level=logging.DEBUG)
 import gzip
 
 import uuid
-import datetime as pydt
 import json
 import bson.json_util as bju
-import arrow
-import argparse
 
-import emission.core.wrapper.user as ecwu
 import emission.storage.timeseries.abstract_timeseries as esta
-import emission.storage.timeseries.timequery as estt
-import emission.storage.decorations.user_queries as esdu
-import emission.storage.timeseries.cache_series as estcs
-# only needed to read the motion_activity
-# https://github.com/e-mission/e-mission-docs/issues/356#issuecomment-520630934
-import emission.net.usercache.abstract_usercache as enua
+#import emission.storage.timeseries.cache_series as estcs
 
-
-def export(loc_time_query, trip_time_query, place_time_query, ma_entry_list, user_id, file_name):
-	ts = esta.TimeSeries.get_time_series(user_id)
-	loc_entry_list = list(estcs.find_entries(user_id, key_list=None, time_query=loc_time_query))
-	trip_entry_list = list(ts.find_entries(key_list=None, time_query=trip_time_query))
-	place_entry_list = list(ts.find_entries(key_list=None, time_query=place_time_query))
-	first_place_extra_query = {'$and': [{'data.enter_ts': {'$exists': False}},
-                                        {'data.exit_ts': {'$exists': True}}]}
-    	first_place_entry_list = list(ts.find_entries(key_list=None, time_query=None, extra_query_list=[first_place_extra_query]))
-    	logging.info("First place entry list = %s" % first_place_entry_list)
+def export(loc_entry_list, trip_entry_list, place_entry_list, ma_entry_list, user_id, file_name, ts):
```

Comment:
I don't think this is the right refactoring approach. `export` doesn't actually do much in this case, or at least, it doesn't do as much as it can. And you are still passing too many parameters. Please refactor.

e.g. something like

```
def export(user_id, file_name, ts, start_ts, end_ts):
```

or to be consistent with https://github.com/e-mission/e-mission-server/pull/824#issuecomment-871790013

```
def export(user_id, ts, start_ts, end_ts):
```

followed by:

```
def save(list, filename):
```

Reply:
Yes, I will put together a REST server prototype and database client. The file handling can then be plugged in from there.

Reply:
Worked on a brief refactoring and checked to make sure all tests and export files were the same as I had confirmed before. 
I few notes: The file_name implementation is still a work in progress as it ties heavily into what we are currently working on. Additionally, I am still not sure what to do about refactoring the ma_entry_list as this is wildly different for both base codes. 
Linked here: https://github.com/e-mission/e-mission-server/pull/824/commits/7ee17f5e5b5b3cfa17d3c9fe30268a86c57f124a